hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -841,6 +841,15 @@ public class HoodieWriteConfig extends HoodieConfig {
       .sinceVersion("1.0.0")
       .withDocumentation("Whether to enable incremental table service. So far Clustering and Compaction support incremental processing.");
 
+  public static final ConfigProperty<Boolean> STREAMING_WRITES_TO_METADATA_TABLE = ConfigProperty
+      .key("hoodie.write.streaming.writes.to.metadata")
+      .defaultValue(false)
+      .markAdvanced()
+      .sinceVersion("1.1.0")
+      .withDocumentation("Whether to enable streaming writes to metadata table or not. With streaming writes, we execute writes to both data table and metadata table "
+          + "using one RDD stage boundary. If not, writes to data table and metadata table happens across stage boundaries. By default "
```

Comment:
nit: generalize the docs on "one RDD stage boundary"

Reply:
hoodie.write.metadata.streaming.write.enabled?